pygmt/modules.py

```diff
@@ -12,6 +12,26 @@
 )
 from .exceptions import GMTInvalidInput
 
+SPECIAL_PARAMS = {
```

Comment:
Perhaps move the variable definition inside the "config" class, since the variable is not used by other functions.

Reply:
Ok, done.